data/aazad20.json

```
@@ -0,0 +1,22 @@
+{
+  "name": "Sarvjeet Singh",
+  "bio": "Software Engineer",
+  "tags" : ["Software Engineer","python","react","Javascript","Tensorflow","Data Science"],
+  "socials": [
+    {
+       "icon": "FaLink",
+      "url": "https://www.linkedin.com/in/sarvjeet-singh-6249551b7/",
```

```suggestion
      "url": "https://www.linkedin.com/in/sarvjeet-singh-6249551b7/"
```
The trailing comma isnot required